arm-0.arm/arm.md

```diff
@@ -4,16 +4,12 @@
 Introduction
 ------------
 
-There is currently a gap in the process for developing Puppet that is
-sometimes critiqued as lack of openness,  frustration over where and how
-to get ideas heard, and the perception that the redmine issue tracker
-functions as a dumping ground for fly by feature requests that sit there
-for years with no, or very little, action or value.
-
-Other open source projects have adopted an enhancement process to deal
-with such issues, and thus, this is an attempt to define ARM - Puppet
-Armatures. This proposal is heavily inspired by the Java
-community's [JEP][JEP1] process.
+Open source projects often adopt an enhancement proposal process to
+encourage design and discussion before complicated development work begins,
+and it's in this spirit that Puppet Armatures came about. This process 
+is heavily inspired by the Java community's [JEP][JEP1] process, but 
+has changed over time to reflect the less formal nature of our 
+community.
 
 In some cases, ARMs are artifacts of the process leading up to one or several
```

Comment:
Since this edit drops the "this is an attempt to define ARM" clause, this first use of "ARM" is without introduction and so might increase cognitive load on first read of the doc.

So I'd suggest an s/Puppet Armatures/Puppet Armatures (often shortened to "ARMs")/ in the new paragraph above.